src/gpuhunt/__init__.py

```diff
@@ -8,6 +8,7 @@
     KNOWN_INTEL_ACCELERATORS,
     KNOWN_NVIDIA_GPUS,
     KNOWN_TPUS,
+    KNOWN_TT_ACCELERATORS,
```

Comment:
We should also export `TTAcceleratorInfo` here (see below, all other `*AcceleratorInfo` are exported)